packages/reactstrap-validation-select/src/AvSelect.js

```diff
@@ -360,6 +372,9 @@ class AvSelect extends AvBaseInput {
             primary: '#3262af',
           },
         })}
+        aria-errormessage={ariaMessage.hasError ? ariaMessage.ariaErrorId : ''}
```

Comment:
- use the touched and hasError vars in this component to check for invalid state.
- We don't want the message to show in the errormessage and placeholder simultaneously. do not add feedbackid unless it's invalid AND a selection has been made (since that is when the placeholder is no longer there)